### PR TITLE
Safety check for cli repo manager deletion

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #921

### CODE REVIEW

File: ga4gh/cli.py [hunk 1]
```diff
@@ -1123,12 +1123,32 @@ def configtest_main(parser=None):
...
+    return userResponse
+
+
 class AbstractRepoCommandRunner(object):
 
     def __init__(self, args):
         self.args = args
         self.repoPath = args.repoPath
         self.repoManager = repo_manager.RepoManager(self.repoPath)
+        if 'force' in args:
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: Why not ``self.force = args.force``?

File: ga4gh/cli.py [hunk 2]
```diff
@@ -1185,7 +1205,8 @@ def __init__(self, args):
         super(DestroyRunner, self).__init__(args)
 
     def run(self):
-        self.repoManager.destroy()
+        def func(): self.repoManager.destroy()
+        self.confirmRun(func, 'the Data Repository')
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: Might as well just have ``self.confirmRun(self.repoManager.destroy, 'the Data Repository')``, right?

**Comment by jeromekelleher** [comment]
Looks good, +1.

File: ga4gh/cli.py (hunk 2 above)
Comment (dcolligan) [inline ga4gh/cli.py]: Yes, but I'd rather keep the congruity with other confirmRun calls (plus, there's no guarantee that we won't add arguments to the destroy signature in the future)

File: ga4gh/cli.py (hunk 1 above)
Comment (dcolligan) [inline ga4gh/cli.py]: That would error for all the commands that did not have a `force` argument, which is most of them ... the error is "no force found in args" or something

Comment (jeromekelleher) [inline ga4gh/cli.py]: Ah I see. OK, looks good.

File: ga4gh/cli.py (hunk 2 above)
Comment (jeromekelleher) [inline ga4gh/cli.py]: Fair enough.

**Comment by jeromekelleher** [comment]
OK, thanks for the updates @dcolligan. Ready to merge after rebase.

**Pull request merged**